kubernetes/redpanda/values.yaml

```diff
@@ -0,0 +1,169 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# values.yaml
+#
+# This file contains values for variables referenced from yaml files in the templates directory.
+#
+# For further information on Helm templating see the documentation at:
+#   https://helm.sh/docs/chart_template_guide/values_files/
+#
+# For an explanation of Redpanda configuration values:
+#   https://vectorized.io/docs/advanced-configuration/
+
+clusterDomain: cluster.local
+
+image:
+  repository: vectorized/redpanda
+  # The imagePullPolicy will default to Always when the tag is 'latest'
+  tag: latest
+
+config:
```

Comment:
Can we link to the docs: https://vectorized.io/docs/configuration/

Reply:
Done